draft-ietf-ace-cwt-proof-of-possession.xml

```diff
@@ -961,9 +949,7 @@
 
     <section title='Acknowledgements' anchor='Acknowledgements' numbered="no">
       <t>
-	Thanks to the following people for their reviews of the specification:
-	Michael Richardson and
-	Jim Schaad.
+	We would like to thank Michael Richardson and Jim Schaad for their reviews of the specification.
```

Comment:
Please revert this change.  The list of reviewers will grow and it's easier to maintain the list when there's one person per line.